### PR TITLE
Remove most usage of ioutil

### PR DESCRIPTION
**Pull request by serprex**
From https://pkg.go.dev/io/ioutil:
As of Go 1.16, the same functionality is now provided by package io or package os, and those implementations should be preferred in new code. See the specific function documentation for details.

Skip 4 invocations of `ioutil.ReadDir` where code uses `FileInfo` beyond what `DirEntry` provides

Largest change is in `internal/databases/postgres/bguploader.go` where a channel took `FileInfo` so I changed it to take `string` since the goroutine was only calling `info.Name()`

### CODE REVIEW

**Review by usernamedt — APPROVED** [review]

**Comment by serprex** [comment]
@mialinx Are you able to approve? This touched a lot of files, seems code ownership rules make usernamedt's approval insufficient

**Comment by serprex** [comment]
@usernamedt @x4m are either of you able to bypass merging being blocked by code owners?

**Review by mialinx — APPROVED** [review]
LGTM: mysql, sqlserver

**Review by perekalov — APPROVED** [review]

**Pull request merged**
